CMakeLists.txt

```diff
@@ -34,7 +34,7 @@ endif()
 
 # Geographiclib installs FindGeographicLib.cmake to this non-standard location
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "/usr/share/cmake/geographiclib/")
-find_package(GeographicLib REQUIRED COMPONENTS STATIC)
+find_package(GeographicLib REQUIRED)
```

Comment:
@AGummyBear Was there any reason for the static linkage previously?

Reply:
My thinking was just that it would result in not needing to install GeographicLib as a dependency unless building, since I think most users are probably running this package as installed. I'm fine with the change to shared though. There was no technical reason to use static.